t/t1092-sparse-checkout-compatibility.sh

```diff
@@ -1254,7 +1254,8 @@ test_expect_success 'sparse index is not expanded: read-tree' '
 	init_repos &&
 
 	ensure_not_expanded checkout -b test-branch update-folder1 &&
-	for MERGE_TREES in "update-folder2"
+	for MERGE_TREES in "update-folder1 update-folder2" \
+		      "update-folder2"
```

Comment:
nit: this whitespace doesn't look right (on GitHub, maybe it's right in the editor).

```
	for MERGE_TREES in "update-folder1 update-folder2" \
			   "update-folder2"
```